test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -163,5 +207,36 @@ private void DeleteProducts(int first_id, int last_id)
                 id += 1;
             }
         }
+
+        private async Task<string> AssertTriggerListenerError(string functionName)
+        {
+            string errorMessage = null;
+            var tcs = new TaskCompletionSource<bool>();
+
+            void OutputHandler(object sender, DataReceivedEventArgs e)
+            {
+                if (errorMessage == null && e.Data?.Contains("Failed to start SQL trigger listener") == true)
```

Comment:
Why not take in the error message we're looking for an only complete when it sees that specific one? That seems simpler since you can just have this method be doing the check instead of each calling test.

Reply:
Can be done. I don't see any issue. It was not clear in the beginning that all the new test methods will end up having the same set of statements.